### PR TITLE
save multiple text states

### PR DESCRIPTION
**Pull request by MixailE**

### CODE REVIEW

File: src/helpers/createPost.ts [hunk 1]
```diff
@@ -13,7 +14,7 @@ import walletStore from 'stores/WalletStore'
 export default async function ({
   text,
   threadId,
-  replyToId,
+  replyToId = ReplyIdDefault,
```
Comment (T-Damer) [inline src/helpers/createPost.ts:17]: Why do we need an optional value? Why don't we make it mandatory?

**Review by T-Damer — APPROVED** [review]

File: src/models/ReplyId.ts [hunk 2]
```diff
@@ -0,0 +1,4 @@
+export const ReplyIdDefault =
```
Comment (T-Damer) [inline src/models/ReplyId.ts:1]: `export default '0x0000000000000000000000000000000000000000000000000000000000000000'`

**Review by smyzin — APPROVED** [review]
Lgtm

File: src/models/ReplyId.ts (hunk 2 above)
Comment (smyzin) [reply src/models/ReplyId.ts:1]: Agree

**Review by T-Damer — APPROVED** [review]

**Pull request merged**